msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -108,7 +108,7 @@ The following table provides RSC application permissions for a team and their ap
 |`ChannelMeetingIncomingAudio.Detect.Group`|Detect incoming audio in channel meetings associated with this team.| Supported | NA |
 |`ChannelMeetingStage.Write.Group`|Show content on the meeting stage of channel meetings associated with this team.| Supported | NA |
 |`InAppPurchase.Allow.Group`|Show and complete in-app purchases for users in this team.| Supported | NA |
-|`LiveShareSession.ReadWrite.Group`| Create and synchronize Live Share sessions for meetings associated with this team. | Supported | NA |
+|`LiveShareSession.ReadWrite.Group`| Allows the app to create and synchronize Live Share sessions for the team and get access related information, such as name and role, about the team's roster and any associated meetings, on behalf of the signed-in user. | Supported | NA |
```

Comment:
For the permissions in the RSC doc page, the ", on behalf of the signed-in user" phrase will not be there.

Reply:
Removed as suggested.